### ISSUE TITLE
LSTM converted model incorrect with batch size > 1

### ISSUE THREAD

**Issue by benschreiber** [onnx/keras-onnx#123]
I have converted a basic LSTM Autoencoder model to ONNX. It gives correct results with batch size = 1 (specifically, only 1 input on axis 0). With batch size > 1, the ONNX model gives incorrect results. This was tested using onnxruntime and keras.

Here is a link to my code: https://gist.github.com/benschreiber/f47e4aadede1188e5e59679784f2db86#file-lstm-py-L228

**Comment by jiafatom** [comment]
This seems to be the same issue as [here](https://github.com/onnx/keras-onnx/issues/94), will fix.

**Comment by jiafatom** [comment]
@benschreiber  just checked in the [fix](https://github.com/onnx/keras-onnx/pull/124). It should work now. Can you pull the latest master and try? Thanks.

**Comment by benschreiber** [comment]
I think this fixed it - thanks! I will continue testing and re-open if I find more issues.
